### PR TITLE
Fetch comments from servlet and display on page

### PR DESCRIPTION
**Pull request by n-wach**
Working:
https://screenshot.googleplex.com/GhdzV8RbFJd

### CODE REVIEW

**Comment by n-wach** [comment]
@billie-cat @f-camp : Wait to review this one until #14 is merged

**Comment by billie-cat** [comment]
I will review this once it is marked as open

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -16,30 +16,33 @@
...
+import com.google.gson.Gson;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** 
-  * Servlet that returns a welcome message to display on Nathan's site. 
-  * TODO: modify this file to handle comments data 
+  * Servlet that returns a hard-coded test comments to display
```
Comment (billie-cat) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:26]:
```suggestion
  * Servlet that returns hard-coded test comments to display
```

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -16,30 +16,33 @@
...
+  * on Nathan's site.
+  * TODO: modify this file to handle comments data
```
Comment (billie-cat) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:28]: Can this TODO be removed?

Comment (n-wach) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:28]: Perhaps, I was gonna wait until I was able to receive comments from users as well

Comment (billie-cat) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:28]: I would make the TODO more descriptive so you know exactly what it entails. Maybe:

TODO: Add support for handling user comments

Comment (n-wach) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:28]: 👍

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (n-wach) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Done

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (n-wach) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Done

**Pull request merged**